pkgs/by-name/ot/otree/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  darwin,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "otree";
+  version = "v0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "fioncat";
+    repo = "otree";
+    rev = version;
+    hash = "sha256-WvoiTu6erNI5Cb9PSoHgL6+coIGWLe46pJVXBZHOLTE=";
+  };
+
+  cargoHash = "sha256-2eFGTpI4yevcxtgtGH+AufMDqsuhOH9CiyXUpjtdTDE=";
+
+  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.IOKit;
```

Comment:
```suggestion
```

Should not be needed now after sdk rework

Reply:
thanks! pushed an update now